src/features/surveys/components/SurveyURLCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import { OpenInNew } from '@mui/icons-material';
+import ZUITextfieldToClipboard from 'zui/ZUITextfieldToClipboard';
+import { Box, Card, Link, Typography } from '@mui/material';
+
+interface SurveyURLCardProps {
+  isOpen: boolean;
+  orgId: string;
+  surveyId: string;
+}
+
+const SurveyURLCard = ({ isOpen, orgId, surveyId }: SurveyURLCardProps) => {
+  return (
+    <Card style={{ padding: 16 }}>
```

Comment:
I believe the card is missing the little green/gray dot in the top right?

Reply:
Yes its missing! I added an indicator now. Here are different variations of `isOpen`

![Screenshot 2023-03-01 at 11 36 51](https://user-images.githubusercontent.com/10441376/222117336-9be731d6-b119-4772-8bec-833b33015711.png)
![Screenshot 2023-03-01 at 11 36 42](https://user-images.githubusercontent.com/10441376/222117344-e207f592-3ef9-46aa-82e3-9d114d1bc7a7.png)